clang/test/CheckedCRewriter/definedType.c

```diff
@@ -109,7 +109,7 @@ baz (*(*lua_test3));
 
 typedef int *StkId;
 void lua_test4(StkId *x) {}
-// CHECK: void lua_test4(_Ptr<_Ptr<int>> x) _Checked {}
+//CHECK: void lua_test4(_Ptr<StkId> x) _Checked {}
```

Comment:
Great! This makes me wonder: Have you tested your change on our large benchmarks, vsftpd at the least?

Reply:
Running it now